openshift/templates/rails-postgresql-persistent.json

```diff
@@ -2,15 +2,15 @@
   "kind": "Template",
   "apiVersion": "v1",
   "metadata": {
-    "name": "rails-postgresql-persistent",
+    "name": "rails-pgsql-persistent",
```

Comment:
I thought we were waiting for the right "label" for display name to use, then we'd update with that. Pending approval/agreement on what that is from @smarterclayton